rotkehlchen/globaldb/upgrades/v2_v3.py

```diff
@@ -519,7 +517,7 @@ def migrate_to_v3(connection: 'DBConnection') -> None:
         );
         """)
 
-        cursor.executescript('PRAGMA foreign_keys=off;')
+        cursor.execute('PRAGMA foreign_keys=off;')
```

Comment:
I'm not sure about this one here. Doesn't it need a commit? I believe not and then it was my mistake